docs/self-hosted-appcircle/configure-server/monitoring.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Monitoring
+description: Monitoring the Appcircle Server service logs and metrics with Grafana.
+tags: [monitoring, logging, Grafana]
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+import SpacetechExampleInfo from '@site/docs/self-hosted-appcircle/configure-server/\_spacetech-example-info.mdx';
+import RestartAppcircleServer from '@site/docs/self-hosted-appcircle/configure-server/\_restart-appcircle-server.mdx';
+import DowntimeCaution from '@site/docs/self-hosted-appcircle/configure-server/\_appcircle-server-downtime-caution.mdx';
+
+## Overview
+
+This document provides instructions on how to access and use the monitoring system implemented for the self-hosted Appcircle server.
+
+The monitoring system is designed to provide visibility into the application logs, enabling you to troubleshoot issues, monitor performance, and gain insights into the application's behavior.
+
+The subsequent sections of this document will guide you through the process of accessing the Grafana web interface, navigating the log data.
+
+:::info
+To access and use the monitoring capabilities, you must be running Appcircle server version `3.15.0` or later.
+:::
+
+## Accessing to Grafana Web UI
```

Comment:
We have an important requirement for this section.

In order to view the container logs on the UI, Appcircle services should be `up` and running `healthy`. So the UI can help diagnose only service issues while the Appcircle server is running. For troubleshooting the boot problems, especially when the `check` command after `up` reports `unhealthy`, CLI tools should be used for accessing the Appcircle service logs.

We should add an explanation for this. (caution box)

___
Note: CLI tools will be worked on in another issue later in the future. For now, we're only mentioning that the UI might not help to detect boot problems.